src/cgnslib.c

```diff
@@ -1434,8 +1517,13 @@ int cg_family_name_write(int file_number, int B, int F,
     cgns_famname *famname = 0;
 
      /* verify input */
-    if (cgi_check_strlen(name) ||
-        cgi_check_strlen(family)) return CG_ERROR;
+    if (cgi_check_strlen(name) /*||
+        cgi_check_strlen(family)*/) return CG_ERROR;
+
+    if ( strlen(family) > 33*20 ) {
```

Comment:
Constants required, whet is 33, what is 20 ?

Reply:
should we have a cgi_check_tree_strlen ?